.circleci/config.yml

```diff
@@ -258,7 +258,12 @@ jobs:
           fingerprints:
             - "08:fc:2b:fb:06:0e:8f:0f:01:1f:28:86:83:89:11:28"
       - run: |
-          if [ "$CIRCLE_REPOSITORY_URL" = "git@github.com:mozilla/fxa.git" ]; then
+          # From https://circleci.com/docs/2.0/env-vars/
```

Comment:
nit: Could use `https://circleci.com/docs/2.0/env-vars/#built-in-environment-variables` to link directly section